tests/test_attributes.py

```diff
@@ -7,6 +7,17 @@
 from app.api import utility as util
 
 
+def test_root(test_app):
```

Comment:
```suggestion
def test_root(test_app, set_test_credentials):
```

Note that we have to set some toy credentials in the test environment (which we do via [this pytest fixture](https://github.com/neurobagel/api/blob/87a0373e08ab6b6e57fa5e9ccfc0f4c9fdae7dd1/tests/conftest.py#L14-L18) in conftest.py)  to ensure we don't get an error when starting up the test app. 

This is what is currently causing the test workflow to fail in the PR checks: https://github.com/neurobagel/api/actions/runs/8363565151/job/22896810427?pr=286#step:6:107